include/nrf_inbuilt_key.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**@file nrf_inbuilt_key.h
+ *
+ * @defgroup nrf_inbuilt_key nRF91 Inbuilt Key Management
+ * @{
+ */
+#ifndef NRF_INBUILT_KEY_H__
+#define NRF_INBUILT_KEY_H__
```

Comment:
I agree that we tend to avoid `nrf_` prefixed libraries, but on the other hand, this API has been around for some time now. Shall we try to retain API compatibility as much as possible instead of adjusting naming to NCS standards?